system/HTTP/RedirectResponse.php

```diff
@@ -93,7 +93,9 @@ public function withInput()
     {
         $session = service('session');
         $session->setFlashdata('_ci_old_input', [
+            // @phpstan-ignore-next-line
             'get'  => $_GET ?? [],
+            // @phpstan-ignore-next-line
             'post' => $_POST ?? [],
```

Comment:
You should do it like this, using identifiers:
```suggestion
            'get'  => $_GET ?? [], // @phpstan-ignore nullCoalesce.variable
            'post' => $_POST ?? [], // @phpstan-ignore nullCoalesce.variable
```